doc/source/whatsnew/v1.1.0.rst

```diff
@@ -215,6 +215,50 @@ MultiIndex
         # Common elements are now guaranteed to be ordered by the left side
         left.intersection(right, sort=False)
 
+- Bug in :meth:`MultiIndex.get_indexer` incorrectly handling use of pad and backfill options.  This is a regression introduced in pandas 0.23.0 (:issue:`29896`)
+
+*pandas >= 0.23, < 1.1.0*
+
+.. code-block:: python
+
+    >>> df = pd.DataFrame({
+    ...     'a': [0, 0, 0, 0],
+    ...     'b': [0, 2, 3, 4],
+    ...     'c': ['A', 'B', 'C', 'D'],
+    ... }).set_index(['a', 'b'])
+    >>> mi_2 = pd.MultiIndex.from_product([[0], [-1, 0, 1, 3, 4, 5]])
+    >>>
+    >>> df.reindex(mi_2, method="backfill")
+          c
+    0 -1  A
+       0  A
+       1  D
+       3  A
+       4  A
+       5  C
+    >>>
+    >>> df.reindex(mi_2, method="pad")
+            c
+    0 -1  NaN
+       0  NaN
+       1    D
+       3  NaN
+       4    A
+       5    C
+
+*pandas 1.1.0*:
+
+.. ipython:: python
```

Comment:
for the new we actually execute it, so this is good here.

however,

show the setupcode (df, mi_2) in an ipython block at the top (as they are the same in both version)

Reply:
done, thanks